source/stylesheets/_refills-header.scss

```diff
@@ -88,14 +88,25 @@
     @include refills-inset;
     border-radius: 30px;
     color: darken(desaturate($refills-logo-color, 50%), 15%);
-    display: inline;
+    display: inline-block;
     font-family: $refills-narrow-sans;
     font-size: 0.8em;
     font-weight: 400;
+    margin-bottom: 4em;
     padding: 0.5em 1.4em;
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    background: #FCFFE0;
```

Comment:
+1, also, prefer use of `background-color` here instead of just `background` per our Guides.